### DISCUSSION TITLE
multiple jk bms on one

### DISCUSSION THREAD

**Discussion by andrewcrosse** [Louisvdw/dbus-serialbattery#585] [General]
hi all
id like to add more than one jk bms.
how do i actually wire the rs485 from each device,
JKBMS 1,RS485 converter(plugged in gps port) wires on output looped to other jkbms's then to usb converter?
or do they all go to separate usb adapters ?
thanks

**Comment by mr-manuel** [comment]
Currently they need all a separate USB adapter. If you are short on USB ports the [Isolated 4 port USB to TTL UART FTDI](https://www.duppa.net/shop/isolated-4-port-usb-to-ttl-uart-ftdi/) adapter from Duppa is nice. Pay attention that it's TTL and not RS485. That means you have to use a DC to DC converter for the VCC output of the GPS port. The other pins can be connected directly.

See also https://github.com/Louisvdw/dbus-serialbattery/discussions/250

**Reply by andrewcrosse** [reply]
Thanks for reply and all the work you've done great job. 
Sorry for probably a simple question will these do ? Only seen people saying they have to be isolated but that means nothing to me. 

https://www.ebay.co.uk/itm/164834557586?mkcid=16&mkevt=1&mkrid=711-127632-2357-0&ssspo=YN7oNj8IRYC&sssrc=2349624&ssuid=e-pWsXmoQFq&var=&widget_ver=artemis&media=COPY

**Reply by mr-manuel** [reply]
1. Did you check the [FAQ](https://louisvdw.github.io/dbus-serialbattery/troubleshoot/faq#which-serial-adaptercable-should-i-use)?
2. Do you have the RS485 adapter for the JKBMS?

**Reply by andrewcrosse** [reply]
Yes have  three rs485 converters from bms. 
Didn't see it in faq I'll take another look. 
Could I use three of those isolated adapters into a 4 way usb extension?

**Reply by mr-manuel** [reply]
> Could I use three of those isolated adapters into a 4 way usb extension?

Yes. See [What can I do, if the BMS communication is unstable?](https://louisvdw.github.io/dbus-serialbattery/troubleshoot/faq/#what-can-i-do-if-the-bms-communication-is-unstable) and [How many USB to serial adapters can I connect?](https://louisvdw.github.io/dbus-serialbattery/troubleshoot/faq#how-many-usb-to-serial-adapters-can-i-connect)
